packages/checkout/sdk/src/postMessageHandler/postMessageHandler.ts

```diff
@@ -6,6 +6,7 @@ export type PostMessageHandlerConfiguration = {
 // todo put these in a types file
 export enum PostMessageHandlerEventType {
   PROVIDER_RELAY = 'PROVIDER_RELAY',
```

Comment:
```suggestion
IMTBL_PROVIDER_RELAY
```

Reply:
since these are post message events, I think we should prefix them all to not collide with other apps